roles/user-management/manage-atlassian-users/README.md

```diff
@@ -11,28 +11,34 @@ This is a sample of a YAML file containing the variables.
 
 ```
 ---
-atlassian_url: https://test.atlassian.net
-atlassian_username: admin
-atlassian_password: admin
-atlassian_users:
-  - firstname: Test
-    lastname: 123
-    password: password123
-    email: test@example.com
-    groups:
+atlassian:
+  url: https://example.atlassian.net
+  username: example
+  password: example
+
+  create:
+    user:
+      - firstname: Test
+        lastname: 123
+	password: example
```

Comment:
This seems "off" - why is the firstname/lastname a list, but the password and email are the same(?) for all of them? ... or was this intended to be a list of users with their own firstname, lastname, password, email, etc.?

Reply:
@oybed firstname,lastname, email and password they are the part of the list. I don't know why it shown this way.  [Here](https://github.com/procrypt/infra-ansible/blob/dea8e80bdef97ab8c4b2e1998bf3a9043a7a864c/roles/user-management/manage-atlassian-users/README.md#role-variables), they are in proper format.